go/packages/dev-mode/main.go

```diff
@@ -58,8 +57,10 @@ func (e *Extension) ExternalExtension() {
 	signal.Notify(sigs, syscall.SIGTERM, syscall.SIGINT)
 	go func() {
 		s := <-sigs
-		cancel()
-		s.String()
+		lib.Info("Received signal: %s %v", s.String(), cancel)
```

Comment:
```suggestion
		lib.Info("Received signal: %s", s.String())
```
`cancel` is a function so it won't print anything meaningful

Reply:
Yeah TBH I was mostly doing this just to get rid of the warning 🤷‍♂️